controllers/postController.js

```diff
@@ -0,0 +1,43 @@
+require('express-async-errors')
+const db = require('../models')
+const Post = db.post
+const cloudinary = require('cloudinary').v2
+require('dotenv').config()
+const fs = require('fs')
+
+
+// this is to configure cloudinary
+cloudinary.config({
+    cloud_name: process.env.CLOUDINARY_NAME ,
+    api_key:  process.env.CLOUDINARY_KEY,
+    api_secret: process.env.CLOUDINARY_SECRET 
+  });
```

Comment:
@ademesojosiah why don't you create a separate file for the cloudinary configuration and import into this file. You know, to make the code more cleaner